index.js

```diff
@@ -116,7 +116,11 @@ class CreateCertificatePlugin {
    * Creates a certificate for the given options set in serverless.yml under custom->customCertificate
    */
   createCertificate() {
-
+    const region = this.options.region ? this.options.region : this.serverless.service.provider.region;
+    if (region.startsWith('cn-')) {
+      this.serverless.cli.log('Skipping certificate summary in Chinese region');
+      return true;
+    }
     this.initializeVariables();
     if (!this.enabled) {
       return this.reportDisabled();
```

Comment:
what about changing the enabled flag instead of  just returning true?

Reply:
What does the return value mean here?

We have certificate creator / summary enabled as we want them in supported regions... But it throws an error in China.

Reply:
what i tried to say here is -> there is already code that logs, that the plugin is disabled if the flag "enabled" is false and exits the plugin. so instead of just returning in the if i might just set the enabled flag to false and continue

Reply:
OK I'll update everything.